src/compile-gate.lisp

```diff
@@ -321,6 +321,13 @@ Keys have the form:
     (:measure qubit)
 ")
 
+(defun reset-apply-operator-cache ()
```

Comment:
Is it wise to `%` this to keep away users who don't read docstrings?

Reply:
I was thinking to just remove it altogether.